droid-swing-ui/src/main/java/uk/gov/nationalarchives/droid/gui/export/ExportDialog.java

```diff
@@ -1,552 +1,547 @@
-    private void initComponents() {
-
-        buttonGroup1 = new ButtonGroup();
-        profileSelectLabel = new JLabel();
-        jScrollPane1 = new JScrollPane();
-        profileSelectTable = new JTable();
-        jPanel1 = new JPanel();
-        cancelButton = new JButton();
-        exportButton = new JButton();
-        RadioOneRowPerFile = new JRadioButton();
-        RadioOneRowPerIdentification = new JRadioButton();
-        cmdEncoding = new JComboBox();
-        jLabel1 = new JLabel();
-
-        setTitle(NbBundle.getMessage(ExportDialog.class, "ExportDialog.title_1")); // NOI18N
-        setAlwaysOnTop(true);
-        setName("exportDialog"); // NOI18N
-
-        profileSelectLabel.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.profileSelectLabel.text_1")); // NOI18N
-
-        jScrollPane1.setPreferredSize(new Dimension(300, 402));
-
-        profileSelectTable.setModel(new DefaultTableModel(
-            new Object [][] {
-
-            },
-            new String [] {
-                "Title 1"
-            }
-        ));
-        profileSelectTable.setRowHeight(20);
-        profileSelectTable.setRowSelectionAllowed(false);
-        profileSelectTable.setShowHorizontalLines(false);
-        profileSelectTable.setShowVerticalLines(false);
-        profileSelectTable.setTableHeader(null);
-        jScrollPane1.setViewportView(profileSelectTable);
-
-        cancelButton.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.cancelButton.text")); // NOI18N
-        cancelButton.setVerticalAlignment(SwingConstants.BOTTOM);
-        cancelButton.addActionListener(new ActionListener() {
-            public void actionPerformed(ActionEvent evt) {
-                cancelButtonActionPerformed(evt);
-            }
-        });
-
-        exportButton.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.generateButton.text")); // NOI18N
-        exportButton.addActionListener(new ActionListener() {
-            public void actionPerformed(ActionEvent evt) {
-                exportButtonActionPerformed(evt);
-            }
-        });
-
-        buttonGroup1.add(RadioOneRowPerFile);
-        RadioOneRowPerFile.setSelected(true);
-        RadioOneRowPerFile.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioOneRowPerFile.text")); // NOI18N
-        RadioOneRowPerFile.setToolTipText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioOneRowPerFile.toolTipText")); // NOI18N
-
-        buttonGroup1.add(RadioOneRowPerIdentification);
-        RadioOneRowPerIdentification.setText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioOneRowPerIdentification.text")); // NOI18N
-        RadioOneRowPerIdentification.setToolTipText(NbBundle.getMessage(ExportDialog.class, "ExportDialog.RadioOneRowPerIdentification.toolTipText")); // NOI18N
-
-        cmdEncoding.setModel(getOutputEncodings());
-        cmdEncoding.setEditor(null);

```

Comment:
Can you please check if this does not make the textArea of the dropdown editable?

Reply:
as discussed on slack, tested, not a problem

Reply:
not used anywhere else in the app either